docs/rules/valid-title.md

```diff
@@ -0,0 +1,65 @@
+# Disallow duplicate setup and teardown hooks (no-duplicate-hooks)
```

Comment:
This title seems to be invalid. 😊 (probably copy-paste mistake)

Reply:
Addressed here: https://github.com/jest-community/eslint-plugin-jest/pull/434 🎉

Reply:
@mrtnzlml Thanks for catching it. I have used the template from other read me to maintain the consistency, but missed to change the title 😬